backend/src/graphql/enum/PasswordEncryptionType.ts

```diff
@@ -0,0 +1,12 @@
+import { registerEnumType } from 'type-graphql'
+
+export enum PasswordEncryptionType {
+  EMAIL = 0,
+  ONE_TIME = 1,
+  GRADIDO_ID = 2,
```

Comment:
remove onetime since we do not implement this now please
```suggestion
  GRADIDO_ID = 1,
```

Reply:
I believe we also need `NO_PASSWORD = 0` for the case that the user has registered but not set the password yet.